### DISCUSSION TITLE
Questions about the profile.png

### DISCUSSION THREAD

**Discussion by mendax1234** [academicpages/academicpages.github.io#2086] [Q&A] [answered]
As you can see from my [website](mendax1234.github.io) now, my profile image is not a circle. My profile.png is not a circle, do I need to make it to a circle for it to be shown as normal on my website?

Thanks

**Comment by rjzupkoii** [comment]
The circle effect around your profile image is done via CSS, so it doesn't have to be a circle and [you can change the style as well](https://github.com/academicpages/academicpages.github.io/wiki/Customizing-the-layout-of-your-site#adjusting-the-avatar).

**Reply by mendax1234** [reply]
Thanks for your reply! Besides, do I need to crop my `profile.png` to a square. Cuz now mine is a rectangular, after using the CSS to make the circle effect, it becomes an oval. So, I am wondering whether I need to `profile.png` to be a square so that after circling it, it won't become an oval.

Besides, where can I find the `.author_avatar`.

Thanks!

**Reply by rjzupkoii** [reply, marked as answer]
Your profile picture can pretty much be any size and if you change the CSS, the image will render based upon the image itself. The `.author_avatar` class can be found in the `_sass/_sidebar.scss` file.

**Reply by mendax1234** [reply]
Thanks. Seems that my config is to circle it, but I get an oval at last. That's very strange and below is my config, can help me with it? Thanks!
```
.author__avatar {
  display: table-cell;
  vertical-align: top;
  width: 36px;
  // set width only, for non-square avatars
  // height: 36px;

  @include breakpoint($large) {
    display: block;
    width: auto;
    height: auto;
  }

  img {
    max-width: 175px;
    border-radius: 50%;

    @include breakpoint($large) {
      padding: 5px;
      border: 1px solid $border-color;
    }
  }
}
```

Edited:
Now seems to work. I forgot what I have done.
